src/streams/projectMilestone.ts

```diff
@@ -0,0 +1,30 @@
+import { AirbyteLogger, AirbyteStreamBase, StreamKey } from "faros-airbyte-cdk";
+import { Dictionary } from "ts-essentials";
+import { LinearClient } from "../client/LinearClient";
+import { ProjectMilestone as Model } from "../client/types";
+
+export class ProjectMilestone extends AirbyteStreamBase {
+  public constructor(
+    protected readonly logger: AirbyteLogger,
+    private readonly client: LinearClient
+  ) {
+    super(logger);
+  }
+
+  public getJsonSchema(): Dictionary<unknown> {
+    return require("../../resources/schemas/projectLink.json");
```

Comment:
```suggestion
    return require("../../resources/schemas/projectMilestone.json");
```

Would be nice if we can auto-generate the path to the schema to avoid typos 🤔

Reply:
I am reluctant to put more effort into this approach. We should move to their low-code integration solution, but currently they have a few bugs there, when i tried it last time.

Reply:
Fair enough.